style.css

```diff
@@ -0,0 +1,11 @@
+@media only screen and (max-width: 800px) {
+    body {
+      background-color: lightpink;
+    }
+  }
+  
+  #main-doc{
+    position:reltive;
```

Comment:
As we write our code we might do it very fast and sometimes we make mistakes. In this line, modifications will not be able to take place because the word `relative` it's not correctly written. Kindly, correct this mistake to see this style reflected in your project.